modules/tinymce/src/themes/silver/main/ts/backstage/Anchors.ts

```diff
@@ -47,8 +47,8 @@ const getInlineDialogAnchor = (contentAreaElement: () => SugarElement<HTMLElemen
     hotspot: lazyAnchorbar(),
     bubble: Bubble.nu(-bubbleSize, bubbleSize, bubbleAlignments),
     layouts: {
-      onRtl: () => [ Layout.southeast ],
```

Comment:
Do we know when/at what width this issue will occur?

Reply:
The replication case is in JIRA